PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -1413,7 +1413,7 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			else if (wParam == NPPMAINMENU)
 				return (LRESULT)_mainMenuHandle;
 			else
-				return NULL;
+				return FALSE;

```

Comment:
Here returning FALSE modifies the semantic of this message.

Reply:
I can change it to `LRESULT{}` or just `0`.